app/teamDetailTemp.html

```diff
@@ -0,0 +1,124 @@
+      <script src="js/site.js"></script>
+</head>
+<body>
+
+<div id = "YourProfile" ng-controller = "ProfileAcc" class="text-center">
+   Please input your data:)<br>Enter Team Name and activity name for load()<br>
+	<form id = "ProfileForm">
+		<label>Team Name:<input type=text ng-model = "FName"></label><br>
+		<label>Team Description:<input type=text ng-model = "FDescription"></label><br>
+		<label>Team Size: <input type=text ng-model = "FSize"></label><br>
+		<label>Which activity: <input type = text ng-model = "FActivity"></label><br>
+		<label>Have who?: <input type = text ng-model = "FMembers"></label><br>
+		<input type = "button" ng-click = "load()" value = "load"> <input type = "button" ng-click = "save()" value = "save">
+	</form>
+	<div id = "testing_ouput">
+		{{ datadebug | json}}
+	</div>
+   
+   </div>
+   
+      <!-- Angular and Angular-route libraries (v1.5.7) -->
+   <script src="lib/angular.min.js"></script>
+   <script src="lib/angular-route.min.js"></script>
+   
+      <!-- Firebase (v3.0.3) and AngularFire (v2.0.0) (after angular library is loaded) -->
+   <script src="lib/firebase.js"></script>     
+   <script src="lib/angularfire.min.js"></script>
+
+     <script>
+	 
+angular.module('teamform-teamDetail-app',['firebase'])
+.controller('ProfileAcc',  ['$scope', '$firebaseObject', '$firebaseArray',function($scope) {
+	
+	//initalizeFirebase();
+	initalizeFirebase();
+	database = firebase.database();
+	
+	$scope.FName = "teamA";
+	$scope.FDescription = "Need C++";
+	$scope.FSize = "";
+	$scope.FActivity = "abs";
+	$scope.datadebug = {
+		name: $scope.FName,
+		description: $scope.FDescription,
+		size: $scope.FSize
+	};
+	
+	$scope.load = function () {
+		alert ("click load :P");
+		FName = $scope.FName;
+		FActivity = $scope.FActivity;
+		if ( FName !== '' ) {
+			
+			var refPath = FActivity + "/team/"+FName;
+			alert (refPath);
+			//This is the way for getting data from firebase
+			//.once("value") <--the value inside value must be value;
+			database.ref(refPath).once("value").then(function (data)
+			{
+				//here use data to get back the data :)				
+				alert(data.val().size);
+				if ( data.val().size != null ) {
+					$scope.FSize = data.val().size;
+				} else {
+					$scope.FSize = "";
+				}
+
+				if ( data.val().teamMembers != null ) {
+					$scope.FMembers = data.val().teamMembers;
+				} else {
+					$scope.FMembers = "";
+				}
+
+				if ( data.val().description != null ) {
+					$scope.FDescription = data.val().description;
+				} else {
+					$scope.FDescription = "";
+				}
+
+				$scope.datadebug = data.val();
+			});
+		}
+	}
+	$scope.save = function () {
+		alert ("click save :P");
+		FName = $scope.FName;
+		FActivity = $scope.FActivity;
+		FMembers = $scope.FMembers;
+		FMemberList = FMembers.split(',');
+		FDescription = $scope.FDescription;
+		FSize = $scope.FSize;
+		if ( FName !== '' && FActivity !=='') {
+			
+			var refPath = FActivity + "/team/"+FName;
```

Comment:
We use FName to locate the firebase here, right? So FName must be unique and mandatory. Or we can make a TeamID for all teams.